pkg/registration/offline_certificate.go

```diff
@@ -41,6 +41,21 @@ type OfflinePayload struct {
 // Reads an offline registration certificate from a read object
 //
 // An error indicates the certificate was probably malformed
+func OfflineCertificateFromBytes(reader io.Reader) (*OfflineCertificate, error) {
```

Comment:
Instead of creating this function, I'd just modify the other with a boolean flag on whether we expect base64 encoding or not. Otherwise, we'd get two functions which are almost identical.